packages/app-store/_components/AppCard.tsx

```diff
@@ -16,24 +16,22 @@ export default function AppCard({
   switchOnClick,
   switchChecked,
   children,
-  setAppData,
```

Comment:
We can call the `setAppData` from the context, so we don't need to pass this down as a prop.